complete/completeprocessing.php

```diff
@@ -1,146 +1,172 @@
 <?php
-	$title = 'Boiler Books';
-	$completeactive = "active";
-	include '../menu.php';
+    $title = 'Boiler Books';
+    $completeactive = "active";
+    include '../menu.php';
 ?>
 
-<?php
-include '../dbinfo.php';
-echo '<div class="container">';
```

Comment:
Lets talk about what's wrong here, first, why are we just echo-ing html.. in the code I moved the div above the `<?php` start and then we don't have to do that anymore